src/style.css

```diff
@@ -1,16 +1,34 @@
 html {
-  font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
+  box-sizing: border-box;
+  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+}
+
+body {
+  height: auto;
+  box-shadow: 2px 2px 2px 2px #000;
+  border-radius: 5px;
+  width: 90%;
+  margin: 20px auto 10px auto;
 }
 
 main {
   padding: 20px;
-  width: 80%;
-  margin: 0 auto;
+  width: 82%;
+  height: 100vh;
+}
+
+#add-new-game {
+  height: auto;
+  width: 100%;
+  box-shadow: 2px 2px 2px 2px #000;
+  padding: 10px;
 }
 
 .rec-scores {
-  float: left;
   width: 40%;
+  margin-top: 10px;
+  padding-bottom: 20px;
+  float: left;
 }
```

Comment:
- [ ] Nice changes you made based on the previous reviewers' comments 👏 . However, the form filed duplicated it self after your new addition. Do take it out, to have the exact replica of the wireframe given 👍 

## Your view:
![image](https://user-images.githubusercontent.com/50990925/231633899-ac7fb17f-8545-46cc-8ca6-60f76552db37.png)

## Wireframe:
![image](https://user-images.githubusercontent.com/50990925/231634139-145835c7-b249-42a6-8f67-75267d0b7c66.png)

Reply:
That is not my project or not mine repository. Watch carefully